Website/src/rendering/src/components/Checkout/AddressCard.tsx

```diff
@@ -0,0 +1,43 @@
+import { DBuyerAddress } from 'src/models/ordercloud/DBuyerAddress';
+
+type AddressCardProps = {
+  address: DBuyerAddress;
+  onClick?: (address: DBuyerAddress) => void;
+  active?: boolean;
+  editable?: boolean;
+  onEdit?: (address: DBuyerAddress) => void;
+};
+
+const AddressCard = (props: AddressCardProps): JSX.Element => {
+  const street2 = props.address.Street2 && <p>{props.address.Street2}</p>;
+
+  const addressClasses = ['address-card'];
+  if (props.active) {
+    addressClasses.push('address-card__active');
+  }
+
+  const editButton = props.editable && (
+    <button onClick={() => props.onEdit(props.address)}>Edit Address</button>
+  );
+
+  const onClick = (address: DBuyerAddress) => {
+    if (props.onClick && typeof props.onClick === 'function') {
```

Comment:
Is the `typeof function` check necessary? The props already define `onClick` as a function.